study-builder/studies/testboston/snippets/question-blood-test-list.conf

```diff
@@ -33,181 +33,7 @@
     ]
   },
   "children": [
-    {
```

Comment:
Pulled these out to separate files so we can reuse them in `followup-survey.conf`.